Author: Evgeny Golyshev <eugulixes@gmail.com>
Description: use rebar instead of rebar3 in tests
 Fix bug which made the Mix tests use only rebar3 when they should
 test Mix with both rebar and rebar3.

--- elixir-lang-1.7.4.orig/lib/mix/test/mix/rebar_test.exs
+++ elixir-lang-1.7.4/lib/mix/test/mix/rebar_test.exs
@@ -162,15 +162,15 @@ defmodule Mix.RebarTest do
     end
 
     test "gets and compiles dependencies for Rebar" do
-      Mix.Project.push(RebarAsDep)
+      Mix.Project.push(RebarAsDepWithEnv)
 
       in_tmp("get and compile dependencies for Rebar", fn ->
         Mix.Tasks.Deps.Get.run([])
         assert_received {:mix_shell, :info, ["* Getting git_rebar" <> _]}
 
         Mix.Tasks.Deps.Compile.run([])
-        assert_received {:mix_shell, :run, ["===> Compiling git_rebar\n"]}
-        assert_received {:mix_shell, :run, ["===> Compiling rebar_dep\n"]}
+        assert_received {:mix_shell, :run, ["==> git_rebar (compile)\n"]}
+        assert_received {:mix_shell, :run, ["==> rebar_dep (compile)\n"]}
         assert :git_rebar.any_function() == :ok
         assert :rebar_dep.any_function() == :ok
 
